core/tests/test_utils.py

```diff
@@ -505,7 +587,12 @@ def save_new_exp_with_states_schema_v0(self, exp_id, user_id, title):
         exp_services.create_and_save_state_id_mapping_model(exploration, [])
 
     def publish_exploration(self, owner_id, exploration_id):
-        """Publish the exploration of given exploration_id."""
+        """Publish the exploration of given exploration_id.
```

Comment:
Publish the exploration with the given exploration_id.

Reply:
Done!